addons/sourcemod/scripting/vsh/abilities/ability_rage_scare.sp

```diff
@@ -15,12 +17,14 @@ public void ScareRage_Create(SaxtonHaleBase boss)
 	boss.SetPropFloat("ScareRage", "Radius", -1.0);
 	boss.SetPropFloat("ScareRage", "Duration", 5.0);
 	boss.SetPropInt("ScareRage", "StunFlags", TF_STUNFLAGS_GHOSTSCARE);
+	boss.SetPropFloat("ScareRage", "Slowdown", 0.25);
```

Comment:
Does this default value affect all classes for a general rage? Even if it doesn't, is there really a purpose to have it here?